modules/codegen/src/main/scala/com/twilio/guardrail/extract/Extractable.scala

```diff
@@ -37,4 +39,9 @@ object Extractable {
       case x: Boolean if x  => EmptyIsNull
       case x: Boolean if !x => EmptyIsEmpty
     })
+  implicit def defaultExtractableList[T]: Extractable[List[T]] =
+    build[List[T]]({
+      case x: Seq[T]       => x.toList
```

Comment:
This seems dangerous. Any new warnings around erasure here?

Reply:
I'd slap on an `Extractable[T]` constraint and fold through the sequence to ensure match